src/app/static/src/tests/components/projects/projectHistory.test.ts

```diff
@@ -77,9 +78,13 @@ describe("Project history component", () => {
         expectTranslated(v.at(4),"Load", "Charger", store);
         expect(v.at(4).find("a").attributes("href")).toBe("");
         expectTranslated(v.at(5), "Delete", "Supprimer", store);
-        expectTranslated(v.at(6),"Copy to a new project", "Copier dans un nouveau projet", store);
-        expect(v.at(6).find("a").attributes("href")).toBe("");
-
+        if (switches.copyProject) {
```

Comment:
All the switching in the tests seems to be on copyProject, should there not also be a test that the shareProject component is/is not rendered?

Reply:
There apparently isn't a test for that at the moment! Can add one.